BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -35,17 +36,30 @@ private static BraintreeClientParams createDefaultParams(Context context, String
                 .getPackageName()
                 .toLowerCase(Locale.ROOT)
                 .replace("_", "") + ".braintree";
-        return createDefaultParams(context, authString, returnUrlScheme);
+        return createDefaultParams(context, authString, returnUrlScheme, UUIDHelper.getFormattedUUID(), IntegrationType.CUSTOM);
     }
 
     private static BraintreeClientParams createDefaultParams(Context context, String authString, String returnUrlScheme) {
+        return createDefaultParams(context, authString, returnUrlScheme, UUIDHelper.getFormattedUUID(), IntegrationType.CUSTOM);
+    }
+
+    private static BraintreeClientParams createDefaultParams(Context context, String authString, String sessionId, @IntegrationType.Integration String integrationType) {
```

Comment:
Maybe a docstring or note clarifying that this is meant for DropIn to enabled proper analytics.

Reply:
We could even rename the func to be more clear that is it's purpose.